app/src/main/java/org/fossasia/openevent/general/settings/LegalFragment.kt

```diff
@@ -0,0 +1,81 @@
+package org.fossasia.openevent.general.settings
+
+import android.os.Bundle
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.preference.Preference
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import com.takisoft.fix.support.v7.preference.PreferenceFragmentCompat
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.utils.Utils
+import java.util.prefs.PreferenceChangeEvent
+import java.util.prefs.PreferenceChangeListener
+
+class LegalFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
+    val PRIVACY_LINK: String = "https://eventyay.com/privacy-policy/"
+    val TERMS_OF_SERVICE_LINK: String = "https://eventyay.com/terms/"
+    val COOKIE_POLICY_LINK: String = "https://eventyay.com/cookie-policy/"
+
+    override fun preferenceChange(evt: PreferenceChangeEvent?) {
+        preferenceChange(evt)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        context?.let { ContextCompat.getColor(it, android.R.color.white) }?.let { view?.setBackgroundColor(it) }
+        return view
+    }
+
+    override fun onCreatePreferencesFix(savedInstanceState: Bundle?, rootKey: String?) {
+        // Load the preferences from an XML resource
```

Comment:
while you are at it, can you remove this comment as well.
Comments should be for stuff that isn't obvious

Reply:
Sure will remove all comments :+1:

Reply:
Updated PR